src/common/mfu_flist.c

```diff
@@ -123,10 +123,10 @@ static size_t list_elem_pack2_size(int detail, uint64_t chars, const elem_t* ele
 {
     size_t size;
     if (detail) {
-        size = 2 * 4 + chars + 0 * 4 + 10 * 8;
+        size = 2 * 4 + chars + 2 * 8 + 0 * 4 + 10 * 8;
     }
     else {
-        size = 2 * 4 + chars + 1 * 4;
+        size = 2 * 4 + chars + 2 * 8 + 1 * 4;
```

Comment:
Since `list_elem_pack2` is wrapped with `#ifdef DAOS_SUPPORT`, maybe we should wrap the size as well?
Maybe we could just keep the size calculation as it was originally, and just do:
```
#ifdef DAOS_SUPPORT
size += 2 * 8; // for obj_id_lo and obj_id_hi
#endif
```

@dsikich What do you think here?

Reply:
@daltonbohning that is fine, it keeps it more clearly separated